tests/utest/oxygen/test_oxygen_cli.py

```diff
@@ -54,10 +55,20 @@ def test_junit_works_on_cli(self):
                      text=True,
                      shell=True)
 
-        for expected in ('<stat pass="69" fail="3">Critical Tests</stat>',
-                         '<stat pass="69" fail="3">All Tests</stat>'):
-            self.assertIn(expected, example.read_text())
-            self.assertIn(expected, actual.read_text())
+        exampleTree = ElementTree.parse(example)
+        exampleRoot = exampleTree.getroot()
+
+        actualTree = ElementTree.parse(actual)
+        actualRoot = actualTree.getroot()
```

Comment:
I propose we do not introduce variables when they are not used:
```suggestion
        example_xml = ElementTree.parse(example).getroot()
        produced_xml = ElementTree.parse(actual).getroot()
```